batch/batch/front_end/front_end.py

```diff
@@ -1134,6 +1135,7 @@ async def _create_jobs(
         spec['job_group_id'] = job_group_id
 
         always_run = spec.pop('always_run', False)
+        n_max_attempts = spec.pop('n_max_attempts', 5)
```

Comment:
If `n_max_attempts` defaults to 20, why is it 5 here?

Reply:
Good spot, this should probably be 20 too